src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
+
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    margin: 0 20px 0 0;
```

Comment:
use margin-left?

Reply:
but I need margin-right, any way you are right that I could put only one margin :)

Reply:
use margin-right property here